pkgs/applications/blockchains/lightwalletd/default.nix

```diff
@@ -0,0 +1,36 @@
+{ buildGoModule, fetchFromGitHub, lib }:
+buildGoModule rec {
+  pname = "lightwalletd";
+  version = "0.4.7";
+
+  src = fetchFromGitHub {
+    owner = "zcash";
+    repo  = "lightwalletd";
+    rev = "a4cc08b89eddd88befc36931422c53e4d4190185";
```

Comment:
```suggestion
    rev = "v${version}";
```

Reply:
I've made this change, however the code [explicitly](https://github.com/zcash/lightwalletd/blob/a4cc08b89eddd88befc36931422c53e4d4190185/Makefile#L22) calls for the commit hash in the `GitCommit` field.